knora/dsplib/utils/xml_upload.py

```diff
@@ -909,7 +909,15 @@ def xml_upload(input_file: str, server: str, user: str, password: str, imgdir: s
 
         resource_bitstream = None
         if resource.bitstream:
-            img = sipi_server.upload_bitstream(os.path.join(imgdir, resource.bitstream.value))
+            try:
+                img = sipi_server.upload_bitstream(os.path.join(imgdir, resource.bitstream.value))
+            except BaseException as err:
```

Comment:
I'm a bit unsure which type of error is best to catch. I tested it by restarting SIPI in the Docker Desktop window while an upload was ongoing. This lead to several error types:
- ConnectionError
- ConnectionResetError,
- BaseException, 
- http.client.RemoteDisconnected
- HTTPConnectionPool

Reply:
OK, now I see. 
For the Exception that is not caught: did you try `except Exception as err:`?